### ISSUE TITLE
CrossMediaManager.Current.Init no parameter

### ISSUE THREAD

**Issue by mmehdidehghani** [martijn00/XamarinMediaManager#485]
I installed Plugin.MediaManager.Forms 0.6 in my Xamarin.Forms project.
When i want to add CrossMediaManager.Current.Init(this) in the MainActivity, I get the this error:
error message : Method int() has 0 parameter(s) but is invoked with 1 argument(s)

Could you please help me?

**Comment by mmehdidehghani** [comment]
error message : Method int() has 0 parameter(s) but is invoked with 1 argument(s)

**Comment by martijn00** [comment]
Probably duplicate of https://github.com/martijn00/XamarinMediaManager/issues/437

You need to target android 9.0

**Comment by EliasSalom** [comment]
I also have the same problem and my target android is 9.0

**Comment by martijn00** [comment]
Now you need to target v10.0.

**Comment by hansmerdijk** [comment]
target to v10.0 does not solve the problem, call init with (this) gives error while building, call init with () does start the emulator but it crashes as soon as it starts up.

**Comment by francoisdutoit** [comment]
> target to v10.0 does not solve the problem, call init with (this) gives error while building, call init with () does start the emulator but it crashes as soon as it starts up.

+1

**Comment by junior-pires** [comment]
I already target to v10.0 but I still facing the issue @martijn00  help plzzz

**Comment by eclipsed4utoo** [comment]
Just confirming that updating to the latest release of this library will break your application on Android. Found out the hard way tonight.

Targetting v9 or v10 doesn't matter. There is no available method `CrossCurrentMedia.Current.Init(Context)`. Using the `Init` method throws an exception at runtime.

I'm going to rollback version by version until it starts working again. I know version 0.8.18 works since that's what I upgraded from.
